PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -528,7 +528,7 @@ export class ParaReportPublishService {
 
                     currentSarifContentString = JSON.stringify(currentSarifContentJson);
                     fs.writeFileSync(currentSarifReport, currentSarifContentString, 'utf8');
-                    tl.uploadArtifact("Container", this.sarifReports[i], this.SARIF_ARTIFACT_NAME);
+                    tl.uploadArtifact("SarifContainer", this.sarifReports[i], this.SARIF_ARTIFACT_NAME);
```

Comment:
In order to avoid uploading two types of file (sarif and cobertura) under the same folder